chilldkg: add API docs for blaming functions (#72)
BlockstreamResearch/bip-frost-dkg: README.md

def participant_blame(blame_state: ParticipantBlameState, cblame: CoordinatorBlameMsg) -> NoReturn
```

> Perform a blame investigation (TODO) for a participant in a ChillDKG session.

Reviewer: ```suggestion
Investigate who is to blame for a failed ChillDKG session.
```

a bit more natural and shorter.

Author: Done

def participant_blame(blame_state: ParticipantBlameState, cblame: CoordinatorBlameMsg) -> NoReturn
```

Perform a blame investigation (TODO) for a participant in a ChillDKG session.

> This function is called when `participant_step2` raises
> `UnknownFaultyParticipantOrCoordinatorError` and allows to further narrow
> down the suspected faulty parties.  It helps narrow down the suspected
> faulty parties by analyzing the blame message provided by the coordinator.

Reviewer: * s/is called/can (optionally) be called
 * the "narrow down" sentence is duplicated :)
 * nit: double space after "parties."
 * I suggest dropping "further". (So far, no narrowing down has taken place.)
 * s/helps narrow down/narrows down 
 * s/narrow down/single out

Here's an attempt to incorporate this:
```suggestion
This function can optionally be called when `participant_step2` raises
`UnknownFaultyParticipantOrCoordinatorError`. It allows to narrow
down the suspected faulty parties by analyzing the blame message
provided by the coordinator.
```

Author: Thanks. Sorry, that paragraph was a mess. Fixed.

> The function does not return a value. Instead, it raises one of two
> exceptions.

Reviewer: nit: I find "s/The function/This function" a tiny bit clearer.

Do you want to primarily say here that the function does not return a value, or that it never returns at all (but raises instead)? Since both are true, and the latter implies the former, we could also write this:

```suggestion
This function does not return normally. Instead, it raises one of two
exceptions.
```

*Arguments*:

> - `blame_state` - The participant's blame state as contained in
>   UnknownFaultyParticipantOrCoordinatorError thrown by

Reviewer: ```suggestion
- `blame_state` - The participant's blame state as contained in
  `UnknownFaultyParticipantOrCoordinatorError` raised by
```

Author: Fixed
